README.md

```diff
@@ -0,0 +1,42 @@
+# keeper
+
+## Setup
+
+This is how to get keeper running on your machine so that you're able to work on it.
+
+First you have to login to clubhouse and generate an API key. You can do this from your account settings -> API key. Save this token somewhere and add it to a new environment variable called `CLUBHOUSE_API_KEY`.
```

Comment:
We should probably add more details on how to do this but for now I think it's better than nothing at least.